backend/src/nodes/nodes/image_filter/high_boost_sharpen.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import HbfTypeDropdown, ImageInput, NumberInput
+from ...properties.outputs import ImageOutput
+from ...utils.image_utils import KernelType
+from . import category as ImageFilterCategory
+
+
+@NodeFactory.register("chainner:image:sharpen_hbf")
+class HbfSharpenNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply sharpening to an image using a high boost filter."
+        self.inputs = [
+            ImageInput(),
+            HbfTypeDropdown(),
+            NumberInput("Amount"),
+        ]
+        self.outputs = [ImageOutput(image_type="Input0")]
+        self.category = ImageFilterCategory
+        self.name = "High Boost Filter"
+        self.icon = "MdBlurOff"
+        self.sub = "Sharpen"
+
+    def run(
+        self,
+        img: np.ndarray,
+        kernel_type: int,
+        amount: int,
+    ) -> np.ndarray:
+        if amount == 0:
+            return img
+
+        delta = np.array([[0, 0, 0], [0, 1, 0], [0, 0, 0]])
+        if kernel_type == KernelType.STRONG:
+            # 8-neighbor kernel
+            kernel = delta - np.array([[1, 1, 1], [1, 1, 1], [1, 1, 1]]) / 9
+        else:
+            # 4-neighbor kernel
+            kernel = np.array([[0, -0.2, 0], [-0.2, 0.8, -0.2], [0, -0.2, 0]])
```

Comment:
Could we make this consistent with the above kernel?

```suggestion
            kernel = delta - np.array([[0, 1, 0], [1, 1, 1], [0, 1, 0]]) / 5
```

This should make it easier to understand how we got those numbers.

Reply:
Sure. I was just optimizing by pre-calculating. Just couldn't do it with the other because the decimals are infinite.

Reply:
This is another case where it would be nice to have some sort of python compiler that can optimize that stuff at compile time

Reply:
"Optimizing"

While it's true that we are wasting cycles calculating constants as runtime, we only save ~100 cycles tops while the actual convolution will take millions of cycles for small images (assuming fast memory allocations and efficient algorithms). It's just not worth the less understandable code. As they say: "premature optimization is the root of all evil."